changelog.d/5-internal/WPB-9065

```diff
@@ -0,0 +1 @@
+Rename effect EmailSmsSubsystem to EmailSubsystem as it has no SMS action anymore
```

Comment:
Feel free to drop this, we never made a release anyway.

Reply:
Ok, I'm removing. But then hopefully we get to merge this PR before we cut a release!